### ISSUE TITLE
Service org.openbmc.records.events should be removed.

### ISSUE THREAD

**Issue by williamspatrick** [openbmc/openbmc#1313]
```
Service org.openbmc.records.events:
└─/org
  └─/org/openbmc
    └─/org/openbmc/records
      └─/org/openbmc/records/events
```

I believe this is now unused, correct?

**Comment by anoo1** [comment]
That's correct, I'll remove it.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/4566> Remove the org.openbmc.records.events service
Closes:  #1313 Service org.openbmc.records.events should be removed.
